src/i18n/en-US/index.ts

```diff
@@ -677,6 +677,11 @@ export default {
       networkStatuses: 'Network Status',
       updatedAgo: 'Updated {time} ago',
       xcmDepositWithdrawalAssets: 'XCM Deposit/Withdrawal Assets',
+      zkEVMNetworkStatuses: 'Astar zkEVM Network Status',
+    },
+    burn: {
+      burn: 'Burn',
+      comingSoon: 'Coming soon',
```

Comment:
I think you don't need to add here as we already have it at [line 135](https://github.com/AstarNetwork/astar-apps/blob/f2e88b24504e350ed160e4d3dc5ae87b226b5708/src/i18n/en-US/index.ts#L135)

Reply:
resolved with [04fff34](https://github.com/AstarNetwork/astar-apps/pull/1239/commits/04fff34cdd250436538cf811729d31db9595b5db)